float32[] point
#Give [x,y]
